src/ts/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+export default function Header (props) {
+  return (
+    <div className="container header">
+      <div className="title">
+        VIKING
+        <div className="title-in">
+          GA
+          <span className="replace">
```

Comment:
I checked out in sketch model that there are two spaces in this caption, so I added them (and added `position: relative` to class `replace` and `position: absolute` to `raido`).